<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectVersionInfoWidget</class>
 <widget class="QWidget" name="ProjectVersionInfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>906</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="grpVersionInfo">
     <property name="title">
      <string>Include version info in project</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="3">
       <widget class="QWidget" name="widget" native="true">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="2" column="1">
          <widget class="QSpinBox" name="spinMinor">
           <property name="maximum">
            <number>999999999</number>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QSpinBox" name="spinBuild">
           <property name="maximum">
            <number>999999999</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Major</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Minor</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Rlease</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QSpinBox" name="spinRelease">
           <property name="maximum">
            <number>999999999</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QSpinBox" name="spinMajor">
           <property name="maximum">
            <number>999999999</number>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Build</string>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0" colspan="3">
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Language:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cbLanguage"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Original filename</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QLineEdit" name="txtCompanyName"/>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="QCheckBox" name="chkSyncProductWithFile">
        <property name="text">
         <string>Sync product with file version</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Internal name</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="txtProductName"/>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="txtFileDescription"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>File version</string>
        </property>
       </widget>
      </item>
      <item row="12" column="0">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Legal trademarks</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Product name</string>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="QLineEdit" name="txtLegalCopyright"/>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Company name</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QLineEdit" name="txtInternalFilename"/>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QCheckBox" name="chkAutoIncreaseBuildNumber">
        <property name="text">
         <string>Auto-increase build number on compile</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLineEdit" name="txtOriginalFilename"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>File description</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Product version</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="txtProductVersion"/>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Legal copyright</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="txtFileVersion"/>
      </item>
      <item row="12" column="1">
       <widget class="QLineEdit" name="txtLegalTrademarks"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>grpVersionInfo</tabstop>
  <tabstop>spinMajor</tabstop>
  <tabstop>spinMinor</tabstop>
  <tabstop>spinRelease</tabstop>
  <tabstop>spinBuild</tabstop>
  <tabstop>chkAutoIncreaseBuildNumber</tabstop>
  <tabstop>chkSyncProductWithFile</tabstop>
  <tabstop>cbLanguage</tabstop>
  <tabstop>txtFileDescription</tabstop>
  <tabstop>txtFileVersion</tabstop>
  <tabstop>txtProductName</tabstop>
  <tabstop>txtProductVersion</tabstop>
  <tabstop>txtOriginalFilename</tabstop>
  <tabstop>txtInternalFilename</tabstop>
  <tabstop>txtCompanyName</tabstop>
  <tabstop>txtLegalCopyright</tabstop>
  <tabstop>txtLegalTrademarks</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
